src/db/spanner/models.rs

```diff
@@ -558,11 +561,17 @@ impl SpannerDb {
     }
 
     pub fn delete_storage_sync(&self, user_id: params::DeleteStorage) -> Result<()> {
-        // XXX: should delete from bso table too
```

Comment:
Sorry, please ignore this bogus comment I added! I forgot to remove it

(bso is interleaved into user_collections w/ ON DELETE CASCADE, so the query removing the row from user_collections automatically does bso deletion for us)